technical-report/sections/00-intro.adoc

```diff
@@ -0,0 +1,54 @@
+
+:sectnums!:
+[[introduction]]
+== Introduction
+
+// Insert introduction here.
+
+OASIS Open Document Format ODF 1.0 (<<iso-iec_26300>>) and Office Open XML (ISO/IEC 29500) are both open document formats for saving and exchanging word processing documents, spreadsheets and presentations. Both formats are XML based but differ in design and scope.
+
+OASIS ODF 1.0 was published by OASIS as an OASIS standard in May 2005. The second edition of ODF 1.0 has been published by OASIS as a committee specification in July 2006 and accepted as an International Standard by ISO (<<iso-iec_26300>>) in December 2006. Office Open XML was first approved in December 2006 by the ECMA International General Assembly as ECMA-376. An updated version was published in November 2008 by ISO (ISO/IEC 29500). The corresponding version, ECMA-376 2nd edition, was published in December 2008.
```

Comment:
We shouldn't copy all the text here, let's only keep 1 paragraph.

Reply:
Re-arranged